pegasus/sites.v3/code.org/i18n/public/international/about.fa-IR.md.partial

```diff
@@ -1,22 +1,18 @@
 ---
-title: درباره
+title: About
```

Comment:
looks like there are quite a few strings reverting to English. Is this expected?